styles/config.scss

```diff
@@ -692,6 +692,16 @@ $dfjump-color-background: gradient(linear, 0% 0%, 100% 0%, from(#5f5f5f80), to(#
 $dfjump-font-size: 22px !default;
 $dfjump-padding: 6px !default;
 
+// ================
+// BHOP JUMP TIMING
+// ================
+$bhopjump-width: 300px !default;
+$bhopjump-height: 20px !default;
```

Comment:
From screenshots this looks a bit tall to me, I'd go with 16px.

Couple other styling comments: 
- Labels would probably look better with at least `font-weight: medium`, maybe `semi-bold`.
- Slightly larger margin between labels and bar

Honestly, I think it should be smaller. I appreciate that it's the same width as the synchronizer bar, but that's something you constantly use to adjust your movement, whilst this is just feedback on each jump. Keep in mind we have a *lot* of possible HUD panels, more coming in the future, so the smaller the better. Almost wonder if we could align it to the right of the crosshair or something.

Reply:
Lmk if my changes are ok. I kept it the same width because it helps me see the jump timing more easily without paying as much attention to the bar. Maybe I should make this configurable?

Reply:
I guess fine for now, future styling stuff can wait, probably for same time as we do hud customizer.